conans/client/command.py

```diff
@@ -2080,7 +2080,9 @@ def _add_common_install_arguments(parser, build_help, lockfile=True):
     parser.add_argument("-r", "--remote", action=OnceArgument,
                         help='Look in the specified remote server')
     parser.add_argument("-u", "--update", action='store_true', default=False,
-                        help="Check updates exist from upstream remotes")
+                        help="Check updates exist for the current reference from upstream remotes."
+                             " It doesn't check for updates for the dependencies, clean the local"
```

Comment:
but --update checks for updates in dependencies too.

Reply:
Ok. We need to improve this message, noone knows how it works, there are lots of issues around this flag. Let's write it.

It appears on many commands:  `test`, `create`, `install`, `info`, `workspace`, `graph lock`. Always the same behavior? What is that behavior?